packages/engine/src/scene/components/ModelComponent.ts

```diff
@@ -143,7 +143,11 @@ function ModelReactor() {
 
   // update src
   useEffect(() => {
-    if (model.src === model.scene?.userData?.src) return
+    if (model.src === model.scene?.userData?.src) {
```

Comment:
this is not correct. if the source is the same, it should be a noop as it is currently

Reply:
Actually this was the reason why the errors where persisting when the original model URL was used.

Using another URL source removes the error when the asset is loaded in [line 170](https://github.com/EtherealEngine/etherealengine/pull/9296/files#diff-daae2524b717be2ba24820c9515621d8714efd6e8d8bed8493853a425dc305cdR170). 
But, when the same source is used the errors are not removed because of the early return.